src/Streamly/Internal/Data/Stream/Concurrent.hs

```diff
@@ -42,35 +43,46 @@ module Streamly.Internal.Data.Stream.Concurrent
     , inspectMode
 
     -- * Combinators
-    -- | Stream combinators using Async channel
+    -- | Stream combinators using a concurrent channel
 
+    -- ** Evaluate
+    -- | Evaluate a stream concurrently using a channel.
     , eval
     , evalWith
+    -- Add unfoldrM/iterateM?
 
+    -- ** Map
+    -- | Use a single channel to evaluate all actions.
     , mapM
     , mapMWith
     , sequence
     , sequenceWith
+    -- Add repeatM/replicateM?
 
-    -- XXX experimental binary ops, move to parallel?
+    -- ** Combine two
+    -- | Use a channel for each pair.
+    -- combine/concur/conjoin
```

Comment:
combine

Reply:
On second thought, leave it as it is. Remove this comment.
`combineWith` is the primitive.